api/Request.json

```
@@ -1522,10 +1522,10 @@
               "notes": "Fragment support added in Opera 46."
             },
             "safari": {
-              "version_added": false
+              "version_added": "10.1"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "10.1"
```

This appears to have not landed in iOS until 10.3 (the iOS versioning varies from desktop Safari in some places): https://trac.webkit.org/browser/webkit/releases/Apple/iOS%2010.3/WebCore/Modules/fetch/FetchRequest.idl

```suggestion
              "version_added": "10.3"
```